test/js/specs/happy.js

```diff
@@ -208,4 +208,206 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
+          crossDevice.verifyCrossDeviceDivider()
+        })
+
+        it('should change the state of the copy to clipboard button after clicking', async () => {
+          driver.get(localhostUrl + `?language=${lang}`)
+          const crossDeviceSyncCopy = documentSelection.copy(lang)
+          goToCrossDeviceScreen()
+          crossDevice.crossDeviceCopyToClipboardBtn.click()
+          crossDevice.verifyCrossDeviceCopyToClipboardBtnChangedState(crossDeviceSyncCopy)
+        })
+
+        it('should display error when number is not provided', async () => {
+          driver.get(localhostUrl + `?language=${lang}`)
+          const crossDeviceSyncCopy = documentSelection.copy(lang)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr('123456789')
+          crossDevice.crossDeviceSendLinkBtn.click()
+          crossDevice.verifyCrossDeviceCheckNumberCorrectError(crossDeviceSyncCopy)
+        })
+
+        it('should display error when number is wrong', async () => {
+          driver.get(localhostUrl + `?language=${lang}`)
+          const crossDeviceSyncCopy = documentSelection.copy(lang)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr('123456789')
+          crossDevice.crossDeviceSendLinkBtn.click()
+          crossDevice.verifyCrossDeviceCheckNumberCorrectError(crossDeviceSyncCopy)
+        })
+
+        it('should send sms and navigate to check your mobile screen ', async () => {
+          const crossDeviceCheckYourMobileCopy = crossDeviceCheckYourMobile.copy(lang)
+          driver.get(localhostUrl + `?language=${lang}`)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr(testDeviceMobileNumber)
+          crossDevice.crossDeviceSendLinkBtn.click()
+          waitForAlertToAppearAndSendSms()
+          crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTitle(crossDeviceCheckYourMobileCopy)
+        })
+
+        describe('cross device check your mobile screen', async () => {
+
+          it('should verify UI elements of the cross device check your mobile screen', async () => {
+            driver.get(localhostUrl + `?language=${lang}`)
+            const crossDeviceCheckYourMobileCopy = crossDeviceCheckYourMobile.copy(lang)
+            goToCrossDeviceScreen()
+            crossDevice.typeMobileNumebr('07495 023357')
+            crossDevice.crossDeviceSendLinkBtn.click()
+            waitForAlertToAppearAndSendSms()
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTitle(crossDeviceCheckYourMobileCopy)
+            if (lang === 'en') {
+              crossDeviceCheckYourMobile.verifyMobileNumberMessage('We’ve sent a secure link to +447495023357')
+            } else {
+              crossDeviceCheckYourMobile.verifyMobileNumberMessage('Hemos enviado un enlace seguro a +447495023357')
+            }
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileMayTakeFewMinutesMessage(crossDeviceCheckYourMobileCopy)
```

Comment:
It would be so much easier to read with less repetition. `crossDeviceCheckYourMobile.verifyBoldMessage()` or `crossDeviceCheckYourMobile.verifySMSWaitMessage()`

Reply:
Improved the naming of the css selectors and test functions.